staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -766,6 +766,7 @@ func describePod(pod *corev1.Pod, events *corev1.EventList) (string, error) {
 		if len(pod.Spec.PriorityClassName) > 0 {
 			w.Write(LEVEL_0, "Priority Class Name:\t%s\n", stringOrNone(pod.Spec.PriorityClassName))
 		}
+		w.Write(LEVEL_0, "Service Account:\t%s\n", pod.Spec.ServiceAccountName)
```

Comment:
naive question, is this guaranteed to not be empty?

Reply:
If `ServiceAccountName` is empty, we may skip printing. If we want to print any case, I think it would be better wrap it like `stringOrNone(pod.Spec.ServiceAccountName)`. This will print `<none>`, instead just empty.

Reply:
Ok, updated to skip printing